packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -83,26 +84,51 @@ export class PieCookieBanner extends LitElement implements CookieBannerProps {
 
     async updated (changedProperties: PropertyValues<this>) {
         // Re-fetch locale when country or language changes
-        if (changedProperties.has('country') || changedProperties.has('language')) {
-            await this._setLocaleBasedOnCountryAndLanguage(this.country, this.language);
+        if (changedProperties.has('language') || changedProperties.has('country')) {
+            await this._setLocaleBasedOnCountryAndLanguage(this.language, this.country);
         }
     }
 
     // Dynamically import locale JSON based on country and language
-    private async _setLocaleBasedOnCountryAndLanguage (country: CountryCode, language: LanguageCode, fallback = false): Promise<void> {
+    private async _setLocaleBasedOnCountryAndLanguage (language: LanguageCode, country: CountryCode): Promise<void> {
         try {
-            this._locale = (await import(`../locales/${language.toLowerCase()}-${country.toLowerCase()}.json`, { assert: { type: 'json' } })).default;
+            const localeString = this._getLocaleString(language, country);
+            this._locale = (await import(`../locales/${localeString}.js`)).default;
```

Comment:
Look at the code a few times and think it isn't possible, but to check - could a malicious string provided find its way into this import?